commit 60c51e693cd44839ae6c97127fa8615a82dd2805
Author: Matthias Klumpp <matthias@tenstral.net>
Date:   Sun Jun 7 16:29:58 2015 +0200

    Search for libsystemd only, instead of the deprecated libsystemd-* libs

diff --git a/configure.ac b/configure.ac
index 655507f..69eba58 100644
--- a/configure.ac
+++ b/configure.ac
@@ -235,7 +235,7 @@ dnl ---------------------------------------------------------------------------
 AC_ARG_ENABLE(systemd, AS_HELP_STRING([--enable-systemd],[enable systemd and logind code]),
 	      enable_systemd=$enableval,enable_systemd=yes)
 if test x$enable_systemd = xyes; then
-	PKG_CHECK_MODULES(SYSTEMD, libsystemd-login libsystemd-journal)
+	PKG_CHECK_MODULES(SYSTEMD, libsystemd)
 	AC_ARG_WITH([systemdsystemunitdir],
 		    AS_HELP_STRING([--with-systemdsystemunitdir=DIR], [Directory for systemd service files]),
 		    [has_systemdsystemunitdir=$with_systemdsystemunitdir],
